pkg/networkmanager/network-interface-members.jsx

```diff
@@ -71,10 +71,7 @@ export const NetworkInterfaceMembers = ({
                         if (val) {
                             with_checkpoint(
                                 model,
-                                function () {
-                                    return member_con.activate(dev)
-                                            .fail(show_unexpected_error);
-                                },
+                                () => member_con.activate(dev).catch(show_unexpected_error),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230722-055749-01048432-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)